wear/src/main/res/values/strings.xml

```diff
@@ -60,4 +60,25 @@
     <string name="persistent_high_for_greater_than">Persistent High for > </string>
     <string name="space_mins">" mins"</string>
 
+    <string name="location_permission_not_approved">&lt;br&gt;&lt;br&gt;&lt;br&gt;WatchFace requires Location permission. Click on this WatchFace or visit Settings &gt; Permissions to approve.</string>
```

Comment:
This is already merged, your version has two copies of each of the new strings and I expect wont compile